runtime/lua/vim/uri.lua

```diff
@@ -84,7 +84,7 @@ local function uri_to_fname(uri)
   end
   uri = uri_decode(uri)
   -- TODO improve this.
-  if is_windows_file_uri(uri) then
+  if is_windows then
```

Comment:
Maybe this `is_windows_file_uri` can be removed now ?

Reply:
Yes.removed.